pom.xml

```diff
@@ -212,6 +212,7 @@
     <module>library</module>
     <module>contrib</module>
     <module>demos</module>
+    <module>sketches</module>
```

Comment:
This is the set of modules that are included by default. I think sketches is a special module which might not be needed by all. 
May be include this as part of the all-modules profile above and remove from here?